### PR TITLE
Mark `cri` log viewer as experimental

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Follow-up to 
- #1847

Due to
- https://github.com/containerd/nerdctl/issues/1956
- https://github.com/containerd/nerdctl/issues/1957

cc @zhaojizhuang

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
@containerd/nerdctl-maintainers Let me release v1.2.0 after merging this one

**Review by djdongjin — APPROVED** [review]
LGTM, thanks

**Review by fahedouch — APPROVED** [review]

**Pull request merged**
